webapp/views.py

```diff
@@ -619,6 +690,15 @@ def advantage_view():
     )
 
 
+def get_contract_period(contract):
+    text_from = contract["contractInfo"]["effectiveFrom"]
+    text_to = contract["contractInfo"]["effectiveTo"]
+    date_from = dateutil.parser.parse(text_from)
+    date_to = dateutil.parser.parse(text_to)
+
+    return "yearly" if (date_to - date_from).days > 31 else "monthly"
```

Comment:
This could be bad for contract nearing their ends

Reply:
Removed.